docs/running-tests/from-local-system.md

```diff
@@ -4,41 +4,30 @@ The tests are designed to be run from your local computer.
 
 ## System Setup
 
-Running the tests requires `python`, `pip` and `virtualenv`, as well as updating
-the system `hosts` file.
-
 WPT requires Python 3.6 or higher.
 
 The required setup is different depending on your operating system.
 
 ### Linux Setup
 
-If not already present, use the system package manager to install `python`,
-`pip` and `virtualenv`.
+If not already present, use the system package manager to install `python3`.
 
 On Debian or Ubuntu:
 
 ```bash
-sudo apt-get install python python-pip virtualenv
+sudo apt-get install python3 pyhton3-pip python3-venv
```

Comment:
`pyhton3-pip` will not be needed, as `python3-venv` will have what it needs to install `pip` in the virtual environment. (Done via `ensurepip` in my testing.)